core/tauri-runtime/src/webview.rs

```diff
@@ -140,9 +140,35 @@ pub trait WindowBuilder: WindowBuilderBase {
   fn max_inner_size(self, max_width: f64, max_height: f64) -> Self;
 
   /// Whether the window is resizable or not.
+  /// When resizable is set to false, native window's maximize button is automatically disabled.
   #[must_use]
   fn resizable(self, resizable: bool) -> Self;
 
+  /// Whether the window's native maximize button is enabled or not.
+  /// If resizable is set to false, this setting is ignored.
+  ///
+  /// See [`Dispatch::set_maximizable`] for details.
+  ///
+  /// [`Dispatch::set_maximizable`]: crate::Dispatch::set_maximizable
```

Comment:
I'd rather embed the documentation here instead of linking to another place.

Reply:
[Fixed](https://github.com/tauri-apps/tauri/pull/6665/commits/51697f033c67ca5e6ef0f105ca83d880228d7e10)